File: Heat/Models/WorkAction.vb
Imports System.ComponentModel.DataAnnotations

Namespace Models

    ''' <summary>
    ''' Rappresenta un intervento di lavoro.
    ''' </summary>
    ''' <remarks></remarks>
    Public Class WorkAction

        Property ID As Integer
        
        Property ActionDate As DateTime
        Property OperationID As Integer
        Property Operation As Operation
        Property AssignedOperatorID As Integer
        Property AssignedOperator As WorkOperator
        Property TypeID As Integer
        Property Type As ActionType
        Property PlantID As Integer
        Property Plant As Plant

    End Class

End Namespace

File: Heat.ConvertedToC#/Models/WorkAction.cs
using System;

namespace Heat.Models
{

    /// <summary>
    /// Rappresenta un intervento di lavoro.
    /// </summary>
    /// <remarks></remarks>
    public class WorkAction
	{

		public int ID { get; set; }

		public DateTime ActionDate { get; set; }
		public int OperationID { get; set; }
		public Operation Operation { get; set; }
		public int AssignedOperatorID { get; set; }
		public WorkOperator AssignedOperator { get; set; }
		public int TypeID { get; set; }
		public ActionType Type { get; set; }
		public int PlantID { get; set; }
		public Plant Plant { get; set; }

	}

}

